app/types/websocket.py

```diff
@@ -307,3 +307,56 @@ async def manage_websocket(
                 connection=websocket,
             )
             raise
+
+
+class OfflineWebsocketConnectionManager(WebsocketConnectionManager):
```

Comment:
I'm not sure why we would need an OfflineWebsocketConnectionManager. We always need a manager which can use either a redis or an inmemory queue.